target/linux/at91/image/sama5.mk

```diff
@@ -54,23 +54,26 @@ endef
 
 define Device/at91-sama5d2_xplained
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D2 Xplained
+  DEVICE_VENDOR := Microchip(Atmel AT91)
```

Comment:
I wonder whether we can't find a nicer solution here.
Either remove the "Atmel AT91" entirely, as the user will have selected at91 explicitly in make menuconfig anyway, or set up alternate names (which I'm not sure about yet).

Reply:
fine with me